### ISSUE TITLE
knowledge_base.load() function issue with WebsiteKnowledgeBase

### ISSUE THREAD

**Issue by DanielChico** [phidatahq/phidata#1011]
Hello. I noticed an inconsistency in the code implementation for the WebsiteKnowledgeBase class.
This image shows de base load function
![image](https://github.com/phidatahq/phidata/assets/106417219/12e9eda3-59ed-4101-ad9e-447c5c12b1ff)
And this is the implementation for WebsiteKnowledgeBase
![image](https://github.com/phidatahq/phidata/assets/106417219/e683038b-6491-42a5-b785-b336e9feb160)
It seems to be using the recreate variable for both recreating and skipping documents

**Comment by ysolanky** [comment]
@DanielChico You're right, the implementation of the `load` function of `WebsiteKnowledgeBase` is different from the Base class. But the logic remains the same

**Comment by DanielChico** [comment]
> @DanielChico You're right, the implementation of the `load` function of `WebsiteKnowledgeBase` is different from the Base class. But the logic remains the same

But is not using the "skip_existing" parameter to skip existing urls instead is using "reacreate" parameter i think this behavior is wrong. For instance, if i don't want to recreate the database table but i want to skip existing urls is not possible.

**Comment by jacobweiss2305** [comment]
Please review phidata\phi\knowledge\website.py
```
        # Given that the crawler needs to parse the URL before existence can be checked
        # We check if the website url exists in the vector db if recreate is False
        urls_to_read = self.urls.copy()
        if not recreate:
            for url in urls_to_read:
                logger.debug(f"Checking if {url} exists in the vector db")
                if self.vector_db.name_exists(name=url):
                    logger.debug(f"Skipping {url} as it exists in the vector db")
                    urls_to_read.remove(url)
```
We have logic for skipping urls if they do exist WITHOUT recreating the database.
